packages/shared/lib/services/sync/run.service.ts

```diff
@@ -557,9 +564,25 @@ export default class SyncRun {
             },
             `syncId:${this.syncId}`
         );
+
+        await BigQueryClient.insert({
```

Comment:
if this insert is already try/catch I would say we don't care about await this here
```suggestion
        void BigQueryClient.insert({
```